### PR TITLE
RES-294: Cache Purge UI

### PR DESCRIPTION
**Pull request by vasturiano**
Let's deploy the current UI so we can test further. We can do any design tweaks and/or fixes after.

Current layout looks like:
screen shot 2018-07-17 at 2 55 31 pm

Page will be available at: https://1.1.1.1/purge-cache/

It's hitting the endpoint at: `/v1/api/purge`

### CODE REVIEW

File: source/pages/partials/purge-cache-base.pug [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    meta(property="og:url" content="https://1.1.1.1")
+    meta(property="og:image" content="https://1.1.1.1/media/card.png")
+
+    meta(name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width")
+
+    link(rel="canonical" href="https://1.1.1.1")
+
+    link(href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700" rel="stylesheet")
+
+    script(async src="https://www.googletagmanager.com/gtag/js?id=UA-10218544-23")
```
Comment (vavrusa) [inline source/pages/partials/purge-cache-base.pug]: Is it necessary to add tracker for cache purge page? I would prefer that it's removed (there's already a tracker on the marketing page)

File: source/pages/partials/purge-cache.pug [hunk 2]
```diff
@@ -0,0 +1,48 @@
+section(data-section="purge-cache")
+  header
+    .logo(alt="1.1.1.1 DNS resolver")
+      include ../../media/logo.svg
+
+    h1.title Purge Cache
```
Comment (vavrusa) [inline source/pages/partials/purge-cache.pug]: The title is "Purge Cache" but the description says "refresh cache". Would it make sense to unify the wording? I would prefer "refresh cache".

File: source/pages/partials/purge-cache.pug [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        option(value="NAPTR") NAPTR
+        option(value="NS") NS
+        option(value="PTR") PTR
+        option(value="SOA") SOA
+        option(value="SPF") SPF
+        option(value="SRV") SRV
+        option(value="SSHFP") SSHFP
+        option(value="TLSA") TLSA
+        option(value="TXT") TXT
+        option(value="ANY") ANY
```
Comment (vavrusa) [inline source/pages/partials/purge-cache.pug]: Resolver doesn't cache or support ANY, can you remove it?

File: source/pages/partials/purge-cache-base.pug (hunk 1 above)
Comment (vasturiano) [reply source/pages/partials/purge-cache-base.pug]: Ok, I removed it. It's also present in the /help page, should we remove that one too?

File: source/pages/partials/purge-cache.pug (hunk 2 above)
Comment (vasturiano) [reply source/pages/partials/purge-cache.pug]: Guess that's an @irtefa question. Should we rename the title to "Refresh Cache"? Keeping in mind the page url is still `/purge-cache`.

File: source/pages/partials/purge-cache.pug (hunk 3 above)
Comment (vasturiano) [reply source/pages/partials/purge-cache.pug]: Yep good call. Removed!

File: source/pages/partials/purge-cache.pug (hunk 2 above)
Comment (irtefa) [reply source/pages/partials/purge-cache.pug]: I like Purge Cache.

**Pull request merged**